### PR TITLE
[Update] Deploying and Managing a Cluster with Linode Kubernetes Engine and the Linode API

### PR DESCRIPTION
**Pull request by leslitagordita**
* remove v4beta from API endpoints
* Update version field to k8s_version
* New sections: add and delete tags

### CODE REVIEW

File: docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md [hunk 1]
```diff
@@ -403,12 +403,86 @@ To update your node pool's node count, send a `PUT` request to the `/lke/cluster
...
+    {{</ output >}}
+
+1. To add new tags to your cluster's existing tags, your request must include a `tags` array with all **previous** and **new** tags. The example request will add the new tags `prod` and `monitoring` to the cluster.
+
+        curl -H "Content-Type: application/json" \
+              -H "Authorization: Bearer $TOKEN" \
+              -X PUT -d '{
+                "tags" : ["ecomm", "blog", "prod", "monitoring"]
+              }' \
+              https://api.linode.com/v4beta/lke/clusters/12345
```
Comment (andystevensname) [inline docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md:448]: Should be v4

File: docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md [hunk 2]
```diff
@@ -403,12 +403,86 @@ To update your node pool's node count, send a `PUT` request to the `/lke/cluster
...
+    {{</ output >}}
+
+1. To delete a tag from your cluster, issue a request with only the tags you would like to keep assigned to your cluster. In the example request, the tags `monitoring` and `prod` are excluded from the `tags` array and so will be deleted from your cluster.
+
+        curl -H "Content-Type: application/json" \
+              -H "Authorization: Bearer $TOKEN" \
+              -X PUT -d '{
+                "tags" : ["ecomm", "blog"]
+              }' \
+              https://api.linode.com/v4beta/lke/clusters/12345
```
Comment (andystevensname) [inline docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md:478]: should be v4

File: docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md [hunk 3]
```diff
@@ -128,7 +128,7 @@ To communicate with your LKE cluster, kubectl requires a copy of your cluster's
 1. Access your LKE cluster's kubeconfig file by sending a `GET` request to the `/lke/clusters/{clusterId}/kubeconfig` endpoint. Ensure you replace `12345` with your cluster's ID that you recorded in the previous section:
 
         curl -H "Authorization: Bearer $TOKEN" \
-          https://api.linode.com/v4beta/lke/clusters/12345/kubeconfig
+          https://api.linode.com/lke/clusters/12345/kubeconfig
```
Comment (andystevensname) [inline docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md:131]: Shouldn't the url be api.linode.com/**v4**/lke?

File: docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md (hunk 1 above)
Comment (leslitagordita) [reply docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md]: fixed!

File: docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md (hunk 2 above)
Comment (leslitagordita) [reply docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md]: fixed!

File: docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md (hunk 3 above)
Comment (leslitagordita) [reply docs/kubernetes/deploy-and-manage-lke-cluster-with-api-a-tutorial/index.md]: fixed!

**Pull request merged**